zproject/computed_settings.py

```diff
@@ -1119,3 +1119,5 @@ def zulip_path(path: str) -> str:
 THUMBOR_KEY = get_secret('thumbor_key')
 
 TWO_FACTOR_PATCH_ADMIN = False
+
+BIG_BLUE_BUTTON_SECRET = get_secret('big_blue_button_secret')
```

Comment:
Put this next to `DROPBOX_APP_KEY` higher up in this file, to better group it with similar settings.